src/bluesearch/entrypoint/database/download.py

```diff
@@ -23,6 +23,14 @@
 
 logger = logging.getLogger(__name__)
 
+STRUCTURE_CHANGE = {
+    "arxiv": datetime(2007, 4, 1),
+    "biorxiv": datetime(2018, 12, 1),
+    "medrxiv": datetime(2020, 10, 1),
+    "pmc": datetime(2021, 9, 1),
+    "pubmed": datetime(2021, 12, 1),  # This should change every year in December
+}
+
```

Comment:
A small naming suggestion, just because later on in `from_month < STRUCTURE_CHANGE[source]` IMO it looks cryptic to compare a month to a structure change.

```suggestion
# Data conventions and formats are different prior to these dates. We
# download only if the starting date is more recent or equal to the
# respective threshold.
MIN_DATE = {
    "arxiv": datetime(2007, 4, 1),
    "biorxiv": datetime(2018, 12, 1),
    "medrxiv": datetime(2020, 10, 1),
    "pmc": datetime(2021, 9, 1),
    "pubmed": datetime(2021, 12, 1),  # This should change every year in December
}

```

Reply:
Done in 9edd58b.